test/gurka/test_landmarks.cc

```diff
@@ -13,16 +13,58 @@ using namespace valhalla::mjolnir;
 const std::string db_name = "landmarks.db";
 const std::filesystem::path file_path = db_name;
 
+namespace {
+valhalla::gurka::map BuildPBF(const std::string& workdir) {
+  const std::string ascii_map = R"(
+      a-----b-----c---d
+      A   B   C   D   E
+    )";
+
+  const gurka::nodes nodes = {
+      {"A", {{"landmark", "yes"}, {"name", ""}, {"amenity", "university"}}},
+      {"B", {{"landmark", "yes"}, {"name", "hai di lao"}, {"amenity", "restaurant"}}},
+      {"C", {{"landmark", "yes"}, {"name", "ke ji lu"}, {"amenity", ""}}}, // no amenity, log error
+      {"D", {{"landmark", "yes"}, {"name", "wan da"}, {"amenity", "cinema"}}},
```

Comment:
also  don't forget to remove the `landmark` in the tests

Reply:
meeting ends in one hour. we can discuss here @kevinkreiser @nilsnolde 
I think the `landmark` key is used to select landmark nodes as you suggested earlier we should only parse landmarks nodes otherwise there are too many. without landmark key how can we achieve that?

Reply:
Only the nodes which have a "amenity" tag having a value of the map you specified in your new header (hospital, restaurant etc) are the nodes we want to save. earlier you were just saving _every_ node. however, the `landmark` tag is useless for us. the gist is, we want to define what we consider a "landmark" and not leave that definition to random OSM mappers. does that make sense now?

Reply:
ah right I understand now. thanks!